tests/functional/adapter/test_session_property.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+set_session_property = "set session query_max_run_time='20s'"
```

Comment:
I would move it directly to model config.

Reply:
`set_session_property` is used at the very end for assertion that specified session property is equal to retrieved session property, so maybe it should stay as it is, please let me know if not